config/install/core.entity_form_display.rdf_entity.spdx_licence.default.yml

```diff
@@ -0,0 +1,53 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.rdf_entity.spdx_licence.field_spdx_licence_id
+    - field.field.rdf_entity.spdx_licence.field_spdx_licence_text
+    - field.field.rdf_entity.spdx_licence.field_spdx_see_also
+    - rdf_entity.rdfentity.spdx_licence
+  module:
```

Comment:
It seems that we've just added dependencies on `link` and `test` modules but `spdx.ino.yml` doesn't reflect the new reality.